python/python-video-generation/tutorial.md

```diff
@@ -0,0 +1,275 @@
+---
```

Comment:
@anddriex the blogpost itself is not needed in the repo. This content is managed by our CMS.